\

```diff
@@ -0,0 +1,13 @@
+DISCORD_TOKEN="NjI3NzcyOTg1ODcyMjIwMTYx.XZBhJw.x4mq2F6iCmAK3Yy8EtexDyYs0_I"
+COMMAND_PREFIX="^"
+# You can find this from "right clicking" on your server and selecting "Copy ID".
+DISCORD_SERVER_ID=
+# You can use ANY channel by "right clicking" on your server and selecting "Copy ID". A #bot channel is recommended.
+DISCORD_BOT_CHANNEL_ID=
+# Generate a firebase service account in the UI, the `gcloud` CLI will generate you the wrong service account.
+# When you download your Firebase service account details, these will be in the downloaded/generated json file.
+FIREBASE_PRIVATE_KEY=""
+FIREBASE_PROJECT_ID=""
+FIREBASE_CLIENT_EMAIL=""
+FIREBASE_DATABASE_URL=""
+GITHUB_TOKEN=""
```

Comment:
This file should not be committed.  

You probably want to reset your Discord token too.